src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -95,7 +94,7 @@ const WorkspaceTabs = ({
 
   const tabs = [
     { name: 'dashboard', link: 'workspace-dashboard' },
-    ...(isFeaturePreviewEnabled('workspace-data-service') || isGoogleWorkspace ? [{ name: 'data', link: 'workspace-data' }] : []),
```

Comment:
Similar comment here. I think we do want to have the data tab for azure workspaces

Reply:
Updated!